frontend/src/states/course-mock.json

```diff
@@ -21,49 +21,63 @@
             "price": 89,
             "location": "Tikkurila, Vantaan opistotalo, 211 Luokka",
             "startDate": "2018-04-19T14:30:00.000Z",
-            "endDate": "2018-04-19T16:00:00.000Z"
+            "endDate": "2018-04-19T16:00:00.000Z",
```

Comment:
What is the purpose of the separate mock data for frontend? Backend already provides mock data and would be good if it wasn't duplicated because that adds up the maintenance burden.

Reply:
This mock data allows for 2 things:
- Front-end ability to move independent from back-end. Remove bottle neck.
- Data to be used in unit-test

Reply:
Yeah, that is a good idea to use it for those but it shouldn't be frontend-specific. The problem with this is again larger than just the frontend. We really need to pay more attention in coordinating everything together so that there's no extra work done serving no purpose. The way I see it, there shouldn't be same mock data duplicated separately for frontend and backend but instead it should be found in a single common place.